### PR TITLE
README: update old repo READMEs

### PR DESCRIPTION
**Pull request by kortschak**

### CODE REVIEW

File: blas/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Gonum BLAS [![Build Status](https://travis-ci.org/gonum/blas.svg?branch=master)](https://travis-ci.org/gonum/blas)  [![Coverage Status](https://coveralls.io/repos/gonum/blas/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/blas?branch=master) [![GoDoc](https://godoc.org/github.com/gonum/blas?status.svg)](https://godoc.org/github.com/gonum/blas)
+# Gonum BLAS [![GoDoc](https://godoc.org/gonum.org/v1/gonum/blas?status.svg)](https://godoc.org/gonum.org/v1/gonum/blas)
 
 A collection of packages to provide BLAS functionality for the [Go programming
 language](http://golang.org)
 
 ## Installation
 ```sh
-  go get github.com/gonum/blas
+  go get godoc.org/v1/gonum/blas/...
```
Comment (vladimir-ch) [inline blas/README.md]: gonum.org

File: blas/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ environment variable to point to the blas installation. More information can be
 
 Then install the blas/cgo package:
 ```sh
-  CGO_LDFLAGS="-L/path/to/OpenBLAS -lopenblas" go install github.com/gonum/blas/cgo
+  CGO_LDFLAGS="-L/path/to/OpenBLAS -lopenblas" go install godoc.org/v1/netlib/blas
```
Comment (vladimir-ch) [inline blas/README.md]: gonum.org

File: blas/README.md [hunk 3]
```diff
@@ -32,12 +32,12 @@ For Windows you can download binary packages for OpenBLAS at
 If you want to use a different BLAS package such as the Intel MKL you can
 adjust the `CGO_LDFLAGS` variable:
 ```sh
-  CGO_LDFLAGS="-lmkl_rt" go install github.com/gonum/blas/cgo
+  CGO_LDFLAGS="-lmkl_rt" go install godoc.org/v1/netlib/blas
```
Comment (vladimir-ch) [inline blas/README.md]: gonum.org

File: blas/README.md [hunk 4]
```diff
@@ -32,12 +32,12 @@ For Windows you can download binary packages for OpenBLAS at
...
 ```
 
 On OS X the easiest solution is to use the libraries provided by the system:
 ```sh
-  CGO_LDFLAGS="-framework Accelerate" go install github.com/gonum/blas/cgo
+  CGO_LDFLAGS="-framework Accelerate" go install godoc.org/v1/netlib/blas
```
Comment (vladimir-ch) [inline blas/README.md]: gonum.org

File: blas/README.md (hunk 1 above)
Comment (kortschak) [inline blas/README.md]: Done.

File: blas/README.md (hunk 2 above)
Comment (kortschak) [inline blas/README.md]: Done.

File: blas/README.md (hunk 3 above)
Comment (kortschak) [inline blas/README.md]: Done.

File: blas/README.md (hunk 4 above)
Comment (kortschak) [inline blas/README.md]: Done.

**Comment by kortschak** [comment]
Ugh. Will fix those.

BTW under the current protocol (experimental use) an approval is dismissed when a new commit is made, so if there are things needing change, click requires change rather than approve.

**Pull request merged**